### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by come-maiz** [joewalnes/resisto.rs#3]
The project has no LICENSE file and some files don't mention a license on the header.
To be clear on the distribution terms, it would be nice if you add one.

**Comment by joewalnes** [comment]
Good point. Because it's a website, I didn't bother, but seeing as it can also be considered an app I guess this makes sense.

Any suggestions for type of license? If you have no preference I'll probably just go with MIT.

**Comment by come-maiz** [comment]
One of my personal goals is to take over the world with free software, so I'd use AGPLv3.
But if that's not what you are after, MIT is good too.
